packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/Modal/Components/MyTemplates.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { sandboxUrl } from '@codesandbox/common/lib/utils/url-generator';
+import history from 'app/utils/history';
+import { useQuery } from '@apollo/react-hooks';
+import {
+  Border,
+  TemplateTitle,
+  TemplateSubTitle,
+  MyTemplates,
+  MyTemplate,
+  Title,
+} from '../elements';
+import { LIST_TEMPLATES } from '../../../../queries';
```

Comment:
We don't currently have the tooling set up to import `.gql` files do we? If not, let's get this added and then co-locate our queries/mutations with the components that use them. IIRC the way we're doing it currently we've got a multi-hundred line queries file and that's not maintainable.

Reply:
I'm not 100% sure. Would be good to add it though!

Reply:
Probably better to save this refactor for a separate PR, as it would touch many files outside the scope we're working in here.

Reply:
Closing as its another scope